README.md.tt

```diff
@@ -61,6 +61,33 @@ All credentials and the `.env` file for this project are stored in Ackama 1Passw
 
     TODO: Ensure that all secrets in this app are stored **only** in Ackama 1Password
 
+## Healthchecks
+
+A healthcheck endpoint is available at http://localhost:3000/healthchecks/all. Healthchecks are 
+configured in `config/initializers/health_checks.rb`. By default, the following health checks are enabled:
+
+* Default (app is running)
+* App version (the git commit that is running)
```

Comment:
Are there any deployment scenarios where this checking of git would fail? Looking at https://github.com/sportngin/okcomputer/blob/master/lib/ok_computer/built_in_checks/app_version_check.rb it seems to read `ENV['SHA']` or the `./REVISION` file in project root.

Reply:
@eoinkelly probably Heroku, right?

Reply:
Heroku exposes several environment variables for tracking the release. These can't be seen using `heroku config`, but can be seen `heroku run bash` and then `env`.

I'll investigate this check and see if we can change where it looks for a release version without extending the whole check.

Reply:
Heroku details that are available:

```
HEROKU_SLUG_COMMIT=4cee8602062e7f1289a7809463bb3488e97beb14
HEROKU_SLUG_DESCRIPTION=Deploy 4cee8602
HEROKU_RELEASE_CREATED_AT=2020-05-08T03:32:37Z
HEROKU_RELEASE_VERSION=v190
```

Reply:
@eoinkelly we could add to the release task in the Procfile to create the REVISION file?

Reply:
Otherwise we'll need to override this check

Reply:
Is this check useful enough to jump through some hoops for it? I'm not sure I've ever found myself **needing** it in the past